driver-core/src/main/java/com/datastax/driver/core/SimpleStatement.java

```diff
@@ -193,6 +194,19 @@ public Object getObject(int i) {
     }
 
     /**
+     * Returns the named values as a {@code Map<String, Object>}.
+     *
+     * @return the named values of this statement.
+     * @throws IllegalStateException     if this statement does not have named values.
+     */
+    public Map<String, Object> getObjectMap() {
```

Comment:
No problem. What you say make good sense, I can take care of this tomorrow and update the PR.